pkg/apiserver/apiserver.go

```diff
@@ -78,6 +78,10 @@ type Mux interface {
 type APIGroupVersion struct {
 	Storage map[string]rest.Storage
 
+	// Root is the APIPrefix under which this is being served.  It can also be the APIPrefix/APIGroup that is being served
```

Comment:
I keep them in "group/version" because we expect the user to Specify `APIVersion: "group/version"` in their json/yaml config file. I agree splitting it into separate fields is preferable but that would require a lot more changes than #14444 does. May we merge #14156 first (it's quite big and will be painful to rebase), and then split the fields in another PR?